server/tables/tables.go

```diff
@@ -220,6 +220,8 @@ type User struct {
 	Email     string
 	ImageURL  string
 
+	GithubToken string
```

Comment:
nit: add a comment that this is used if set otherwise the group one is used?

Reply:
There's currently no fallback behavior, but I added a comment.